cfgov/tccp/jinja2/tccp/cards.html

```diff
@@ -14,6 +14,7 @@
 {% block javascript scoped %}
 {{ super() }}
 <script src="{{ static( 'js/routes/on-demand/filterable-list-controls.js' ) }}"></script>
+<script src="{{ static( 'apps/tccp/js/index.js' ) }}"></script>
```

Comment:
I tried using our standard [`jsl`](https://github.com/cfpb/consumerfinance.gov/blob/4ed497883c7a17b66edda4c94f6b9584857a9e37/cfgov/regulations3k/jinja2/regulations3k/landing-page.html#L47-L50) loading function here but I couldn't get the htmx library to load correctly. It might be incompatible with our loading function. Further exploration needed.